experimental/packages/opentelemetry-sdk-node/README.md

```diff
@@ -115,9 +115,25 @@ Configure a resource. Resources may also be detected by using the `autoDetectRes
 
 ### resourceDetectors
 
-Configure resource detectors. By default, the resource detectors are [envDetector, processDetector].
+Configure resource detectors. By default, the resource detectors are [envDetector, processDetector, hostDetector].
 NOTE: In order to enable the detection, the parameter `autoDetectResources` has to be `true`.
 
+You can also use the environment variable `OTEL_NODE_RESOURCE_DETECTORS` to enable only certain detectors, or completely disable them:
+
+- `env`
+- `host`
+- `os`
+- `process`
+- `serviceinstance`
+- `all` - enable all resource detectors above
```

Comment:
I think we should let people know that we may add new detectors which will be included in this scope. Maybe we can avoid some surprises this way. :slightly_smiling_face:

Reply:
do you mean a warning for people that will use or a warning for developers to not forget to add here?
The reason why I added the `above` was because we do have several other detectors that are not included on this list, so I wanted to make sure people don't think all detectors are being counted here and clarifying the only ones that do. 
Do you prefer:
1 - Change this to something `enable all resources detectors above (other detectors might also be included)`
2 - Add a comment on the part of the code that adds the list, with a reminder to update this doc

I'm leaning to option 2 because the first one doesn't help the user to know what is being detected, and I want to make it very obvious

Reply:
> 2 - Add a comment on the part of the code that adds the list, with a reminder to update this doc

I think having a comment there is also a good idea, I had not thought of that :slightly_smiling_face: 

I was thinking something like:
```suggestion
- `all` - enable all resource detectors above
  - **NOTE:** future versions of `@opentelemetry/sdk-node` may include additional detectors that will be covered by this scope.
```

In case users enable `all` as they're okay with the detectors listed above *today*, but they don't consider that the list can grow as we add more features *tomorrow*.

Reply:
ahh I see what you mean! Great point! Committed your suggestion!